apps/website/src/components/tech/Network.tsx

```diff
@@ -79,6 +79,10 @@ const nodeTypes: {
     container: "border-yellow-700",
     eyebrow: { name: "Server", color: "text-yellow-700" },
   },
+  controlunit: {
+    container: "border-red-700",
+    eyebrow: { name: "Camera Control Unit", color: "text-yellow-700" },
```

Comment:
```suggestion
    eyebrow: { name: "Camera Control Unit", color: "text-red-700" },
```
I would think we'd want the text to match the border, to be consistent with everything else?